src/prognostic_equations/edmfx_sgs_flux.jl

```diff
@@ -221,16 +437,39 @@ function edmfx_sgs_diffusive_flux_tendency!(
             @. Yₜ.c.ρq_tot -= ᶜρχₜ_diffusion
             @. Yₜ.c.ρ -= ᶜρχₜ_diffusion
         end
+        if (
+            p.atmos.moisture_model isa NonEquilMoistModel &&
+            p.atmos.precip_model isa Microphysics1Moment
+        )
+            # TODO - we don't want that for rain and snow most likely. Have a parameter
```

Comment:
We have a parameter now if you want to use it.

Reply:
Added. Btw, is there any corresponding implicit solver Jacobian term that needs to be updated?

Reply:
Right now we don't include any environment diffusion in implicit solver